NetworkConfigurations/RoutingAndForwarding/OSPFv2.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Open Shortest Path First Version 2
+grand_parent: Network Configuration
+parent: Routing and Forwarding
+nav_order: 3
+layout: default
+---
+
+# Open Shortest Path First Version 2 (OSPFv2)
+
+OSPFv2 is a widely used interior gateway protocol (IGP) for routing Internet Protocol (IP) packets within a single routing domain, which can be an autonomous system. It uses a link-state routing algorithm and provides efficient and scalable routing in large and complex network topologies. For more information, please refer- [OSPFv2](https://datatracker.ietf.org/doc/html/rfc2328).
+
+This document provides detailed instructions for installing and configuring OSPFv2 on DENT devices using FRR (Free Range Routing).
+
+## Installation of FRR
+
+To install FRR (Free Range Routing) with OSPF support on DENT Devices, follow the steps below:
+
+### Installation using APT:
+
+```
+curl -s https://deb.frrouting.org/frr/keys.gpg | sudo tee /usr/share/keyrings/frrouting.gpg > /dev/null
```

Comment:
Korel is working on BGP testing which requires installing FRR. Are you guys setting up the FRR the same way?

Reply:
Yes, we are setting it up the same way.